uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/calendar-model.js

```diff
@@ -20,14 +20,15 @@ define([
          * as a list of days and their type
          *
          * @param {string/int/Array} contactId can be also an array for multiple contacts
```

Comment:
Sorry, I'm afraid that changing either the implementation or the places where the method is called will cause a chain of changes that I just can't afford in this PR. It is a good point, but I won't risk increasing the scope of the PR